bokehjs/src/coffee/models/plots/plot_canvas.coffee

```diff
@@ -170,6 +170,12 @@ class PlotCanvasView extends Renderer.View
       # Just need to wait a small delay so container has a width
       _.delay(@resize, 10)
 
+    # Re-render when dpi changes (modern browsers emit a resize event on zoom)
+    check_resize_need_render = () ->
+      if @canvas_view.need_render()
+         @request_render()
+    $(window).on("resize", check_resize_need_render.bind(this))
```

Comment:
I've started [using jquery](https://github.com/bokeh/bokeh/pull/4343/files#diff-f9194b68ef1a3a51bbb8f5b03193b403R91) proxy for this kind of thing: `$(window).on("resize", $.proxy(@resize, @))`

My default is an assumption that jquery's thought about this for me. As I haven't thought about it, do you have any opinions - then we can be consistent.